src/extend-zod.ts

```diff
@@ -1,6 +1,18 @@
+import { ZodDate } from "zod";
```

Comment:
double import here

Reply:
should be fine
https://stackoverflow.com/questions/48951687/override-an-export-from-another-library-es6